octopoes/octopoes/core/service.py

```diff
@@ -47,6 +47,31 @@
 settings = Settings()
 
 
+class BitCache:
+    def __init__(self):
+        self.bit_cache: dict[int, tuple[list[OOI], datetime]] = {}
+
+    def __getitem__(self, key: int):
+        return self.bit_cache[key][0]
+
+    def push(self, key: int, data: list[OOI], t: datetime = datetime.now()) -> None:
+        self.bit_cache[key] = (data, t)
```

Comment:
Two things:
- `t` -> `time` or `timestamp`
- `time` will always be the same value once it's imported and if it's not passed as an argument when calling this function. A Pythonic way to handle this:
```suggestion
    def push(self, key: int, data: list[OOI], time: datetime | None = None) -> None:
        if time is None:
            time = datetime.now()
        self.bit_cache[key] = data, time
```

Reply:
Addressed in a9ff441948996c1e6d66a3464c5d5e2f95088da9